### PR TITLE
Edit media change file

### PR DESCRIPTION
**Pull request by spennythug**

### CODE REVIEW

File: www/templates/html/scripts/uploadScript.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                //Bad response, fail for now.
+                return;
+            }
+            
+            if (response.result !== 'complete') {
+                //did not complete
+                return;
+            }
+
+            console.log("DONE YO")
```
Comment (mfairchild365) [inline www/templates/html/scripts/uploadScript.js]: Probably don't need this anymore. ;)

Comment (spennythug) [inline www/templates/html/scripts/uploadScript.js]: This is completely necessary.

File: src/UNL/MediaHub/Manager/PostHandler.php [hunk 2]
```diff
@@ -346,10 +346,24 @@ function handleFeedMedia()
             throw new Exception('You must select a feed for the media', 400);
         }
 
+
+
```
Comment (mfairchild365) [inline src/UNL/MediaHub/Manager/PostHandler.php]: Some unnecessary new lines.  Picky, I know.

File: www/manager/templates/css/forms.css [hunk 3]
```diff
@@ -101,6 +101,13 @@ iframe * {margin:0px}
     display: block;
     cursor: pointer;
 }
+
+.mh-upload-box.mh-upload-box-small {
+
+    padding: 20% 20px 80% 20px;
+
```
Comment (mfairchild365) [inline www/manager/templates/css/forms.css]: Some unnecessary new lines.  Picky, I know.

File: www/templates/html/scripts/uploadScript.js (hunk 1 above)
Comment (spennythug) [inline www/templates/html/scripts/uploadScript.js]: I'm glad I stopped using swear words my logs...

File: www/manager/templates/css/forms.css (hunk 3 above)
Comment (spennythug) [inline www/manager/templates/css/forms.css]: white space makes files easier to read. Iwillfightchu.

**Comment by mfairchild365** [comment]
Looks great!  My only remaining thought is about the presentation of the filename, which can overflow the content area.  Do we even need to show them the filename?

screen shot 2015-01-23 at 4 53 35 pm

**Pull request merged**